Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/Detector.h

```diff
@@ -306,7 +308,7 @@ class Detector : public o2::Base::Detector
     GeometryHandler *mGeometryHandler;
     MisalignmentParameter *mMisalignmentParameter;
 
-    V1Layer **mGeometry;   //! Geometry
+    V3Layer **mGeometry;   //! Geometry
```

Comment:
@sawenzel  I agree with you, Sandro...
In fact, if we look at the data member declarations inside Detector.h, we will find about 15 similar cases more:  raw pointers to some arrays allocated later with "new".   They would call for a similar improvement as well.    Moreover, all those array are actually of a known size, which is 7 (the number of layers, frozen now).  Potentially, we could use also this fact to simplify our geometry code even more.

Doing it in a clean way may take quite some additional time.  The changes would be important from the code maintenance point of view, and at the end, they would still result in the same "geometry tree".

So, I think, we should do it in a separate (maybe not even immediately next) PR.